### DISCUSSION TITLE
Non Profit Organization established

### DISCUSSION THREAD

**Discussion by trafficpest** [teambtcmap/btcmap.org#61] [General]
I was curious if BTCmap.org is an official non profit for fund raising especially in fiat. I have experience in forming a non profit and could do the paper work and pay the small formation fees for you guys (forming in the United States). I have done California but the state paperwork is similar elsewhere and the federal is all the same. 

This would allow you to open a bank account and merchant account to take fiat donations and be non taxable income. let me know if you have done this, or would like to.

**Comment by secondl1ght** [comment]
BTC Map is not currently any form of official non-profit although we might have to form one in the future - not for the reasons of accepting fiat (I dont think it would make sense to ever accept fiat when we can accept bitcoin) but for things like accepting grants if we get awarded with one. I think some of the grant organizations require something like this.

I dont know much about this but I am not sure if we would want to do it in the US - there are probably other jurisdictions that are more bitcoin-friendly I think. But @dadofsambonzuki has been the one looking into this. I will share this post with him and maybe you guys can setup a chat sometime if it makes sense to do so. Thanks for sharing your knowledge on this!

**Comment by dadofsambonzuki** [comment]
Hey @trafficpest - thanks for getting in touch with the details about the non-profit and for your kind offer. As @secondl1ght said, this is something I'm looking at as part of a wider analysis of incorporation and jurisdiction.

I'll re-open this ticket as and when we look at the non-profit side in more detail.
